src/components/Nav/Nav.js

```diff
@@ -1,7 +1,106 @@
-import React from 'react';
+import { React, useState, useEffect } from 'react';
+import { Link } from 'react-router-dom';
+import './Nav.scss';
 
 const Nav = () => {
-  return <div>Nav</div>;
+  const [navCategory, setNavCategory] = useState([]);
+  const [isHovering, setIsHovering] = useState(0);
+  const [myHovering, setMyHovering] = useState(0);
+
+  useEffect(() => {
+    fetch('./data/Nav.json', {
```

Comment:
public 하위의 절대경로를 나타내기 때문에 `.` 없어도 됩니다!
```suggestion
    fetch('/data/Nav.json', {
```

Reply:
그리고 nav의 category list 데이터를 실제 서버에서 받아오는 방식으로 백엔드 팀원과 얘기가 된 부분인가요??

Reply:
네! 얘기 된 부분입니다